packages/jupyter-ai/jupyter_ai/chat_handlers/fix.py

```diff
@@ -64,22 +67,36 @@ class FixChatHandler(BaseChatHandler):
 
     def __init__(self, *args, **kwargs):
         super().__init__(*args, **kwargs)
+        self.prompt_template = None
 
     def create_llm_chain(
         self, provider: Type[BaseProvider], provider_params: Dict[str, str]
     ):
         unified_parameters = {
+            "verbose": True,
             **provider_params,
             **(self.get_model_parameters(provider, provider_params)),
         }
         llm = provider(**unified_parameters)
-
         self.llm = llm
-        # TODO: migrate this class to use a LCEL `Runnable` instead of
-        # `Chain`, then remove the below ignore comment.
-        self.llm_chain = LLMChain(  # type:ignore[arg-type]
-            llm=llm, prompt=FIX_PROMPT_TEMPLATE, verbose=True
-        )
+        prompt_template = llm.get_chat_prompt_template()
```

Comment:
The prompt template should be created from the `FIX_STRING_TEMPLATE` defined above. Right now, this implementation is a copy of `DefaultChatHandler`.

Reply:
Thanks, the `/fix` prompt template with its attributes is now being used.